script/Base/post-deployment/01_configureVRFSubscription.s.sol

```diff
@@ -36,21 +40,67 @@ contract ConfigureVRFSubscription_Base is BatchScript {
             perpetualMint
         );
 
-        addToBatch(supraVRFDepositContract, addContractToWhitelistTx);
+        // This version is makes possible to directly make the gnosis safe execute the transactions required by broadcasting the transactions for each signer.
```

Comment:
There will never be an instance where a single `.env` file holds multiple multisig signer PKs - it seems this is in stark contrast to general security practices.

Please remove the commented code.